pkg/csconfig/database.go

```diff
@@ -133,23 +138,51 @@ func (d *DatabaseCfg) ConnectionString() string {
 
 		connString = fmt.Sprintf("file:%s?%s", d.DbPath, sqliteConnectionStringParameters)
 	case "mysql":
+		params := url.Values{}
+		params.Add("parseTime", "True")
+
+		tlsConfig := &tls.Config{}
+
 		if d.isSocketConfig() {
-			connString = fmt.Sprintf("%s:%s@unix(%s)/%s?parseTime=True", d.User, d.Password, d.DbPath, d.DbName)
+			connString = fmt.Sprintf("%s:%s@unix(%s)/%s", d.User, d.Password, d.DbPath, d.DbName)
 		} else {
-			connString = fmt.Sprintf("%s:%s@tcp(%s:%d)/%s?parseTime=True", d.User, d.Password, d.Host, d.Port, d.DbName)
+			connString = fmt.Sprintf("%s:%s@tcp(%s:%d)/%s", d.User, d.Password, d.Host, d.Port, d.DbName)
 		}
 
 		if d.SSLMode != "" {
-			connString = fmt.Sprintf("%s&tls=%s", connString, d.SSLMode)
+			// This will be overridden if a CA or client cert is provided
+			params.Set("tls", d.SSLMode)
 		}
 
 		if d.SSLCACert != "" {
-			connString = fmt.Sprintf("%s&tls-ca=%s", connString, d.SSLCACert)
+			caCert, err := os.ReadFile(d.SSLCACert)
+			if err != nil {
+				return "", fmt.Errorf("failed to read CA cert file %s: %w", d.SSLCACert, err)
+			}
+			tlsConfig.RootCAs = x509.NewCertPool()
```

Comment:
We might want to do what we do for mutal TLS and get the system pool and if nil we then create an empty pool?

incase a user want to put CA in trustedroots and then use clientkey and password.